aiohttp/web_runner.py

```diff
@@ -238,6 +242,24 @@ def sites(self) -> Set[BaseSite]:
 
         self._server = await self._make_server()
 
+    async def serve_forever(self) -> None:
+        if self._serving_forever_fut is not None:
+            raise RuntimeError(
+                'Concurrent calls to serve_forever() are not allowed'
+            )
+        if self._server is None:
+            raise RuntimeError('Call setup() first')
```

Comment:
Any reason you don't just call `setup()` here? The name `serve_forever()` kinda implies that it'll start serving things by itself. Or perhaps it needs a better name.

Reply:
`setup()` doesn't serve, need to register `TCPSite` first.
The initialization procedure requires explicit steps with specific parameters passing.
This is probably my design mistake, the forecast ability didn't work well here.
So, `serve_forever()` cannot start serving.
I prefer keeping this name because it looks good and asyncio already utilized it for very similar purpose.